.github/workflows/kickstart-tests.yml

```diff
@@ -0,0 +1,150 @@
+       TEST_JOBS: 16
+       REPOSITORY: "http://dl.fedoraproject.org/pub/fedora/linux/releases/36"
+       PLATFORM: "fedora_rawhide"
+    steps:
+      # self-hosted runners don't do this automatically; also useful to keep stuff around for debugging
+      # need to run sudo as the launch script and the container create root/other user owned files
+      - name: Clean up previous run
+        run: |
+          sudo podman ps -q --all --filter='ancestor=kstest-runner' | xargs -tr sudo podman rm -f
+          sudo podman volume rm --all || true
+          sudo rm -rf * .git
+
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.pr-info.outputs.sha }}
+          path: permian
+          fetch-depth: 0
+
+      - name: Rebase to current ${{ env.TARGET_BRANCH }}
+        working-directory: ./permian
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git log --oneline -1 origin/${{ env.TARGET_BRANCH }}
+          git rebase origin/${{ env.TARGET_BRANCH }}
+
+      - name: Clone tplib repository
+        uses: actions/checkout@v3
+        with:
+          repository: rhinstaller/tplib
+          path: tplib
+
+      - name: Clone kickstart-tests repository
+        uses: actions/checkout@v3
+        with:
+          repository: rhinstaller/kickstart-tests
+          path: kickstart-tests
+
+      - name: Generate tplib test cases
+        working-directory: ./kickstart-tests
+        run: scripts/generate-testcases.py -t ./testlib/test_cases/kstest-template.tc.yaml.j2 . -o ./testlib/test_cases
+
+      - name: Ensure http proxy is running
+        run: sudo kickstart-tests/containers/squid.sh start
+
+      - name: Configure Permian event values
+        id: set_event_values
+        run: |
+          set -eux
+          echo "::set-output name=installation_tree::${REPOSITORY}/Everything/x86_64/os/"
+          echo "::set-output name=modular_url::${REPOSITORY}/Modular/x86_64/os/"
+          echo "::set-output name=platform::${PLATFORM}"
+
+      - name: Create Permian settings file
+        working-directory: ./permian
+        run: |
+          cat <<EOF > settings.ini
+          [kickstart_test]
+          kstest_local_repo=${{ github.workspace }}/kickstart-tests
+          [library]
+          directPath=${{ github.workspace }}/kickstart-tests/testlib
+          [github]
+          pull-request=${{ needs.pr-info.outputs.pr_number }}
+          repository=${{ github.repository }}
+          token=${{ secrets.GITHUB_TOKEN }}
+          EOF
+
+      - name: Run kickstart tests in container
+        working-directory: ./permian
+        run: |
+          sudo --preserve-env=TEST_JOBS \
+          PYTHONPATH=${PYTHONPATH:-}:${{ github.workspace }}/tplib \
+          ./pipeline --debug-log permian.log \
+            --settings settings.ini \
+            --override workflows.dry_run=False \
```

Comment:
This is default

Reply:
Okay, I was leaving it in our workflows as a kind of hint, but I think it shouldn't be here.